packages/flutter_goldens_client/lib/skia_client.dart

```diff
@@ -418,12 +418,18 @@ class SkiaGoldClient {
   ///
   /// Example TraceID for Flutter Gold:
   ///   ',CI=cirrus,Platform=linux,name=cupertino.activityIndicator.inprogress.1.0,source_type=flutter,'
```

Comment:
nit: this comment is no longer up to date - the trace id is the hex-encoded md5 hash